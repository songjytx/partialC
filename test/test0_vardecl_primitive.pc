void main(){
	int a = -8;
	printi(a);
	
	float b = 1.2;
	printf(b);

	bool c = true;
	bool d = false;
}