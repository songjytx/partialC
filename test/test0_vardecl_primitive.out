-8
1.200000
